scripts/deploy_local.sh

```diff
@@ -1,5 +1,11 @@
 #!/bin/sh
 
+if [ "$1" = "" ]
```

Comment:
the idea here is that the user needs to pass in their own wasm address? if so, the message could be a bit more descriptive, and/or we should list something in the README to describe what the user should pass into this script.